.travis.yml

```diff
@@ -1,6 +1,6 @@
 os: osx
 language: swift
-osx_image: xcode8
+osx_image: xcode9
 
 branches:
     only:
```

Comment:
Can you also please update the destination to 'platform=iOS Simulator,name=iPhone 8,OS=11.0' in line 10? Thanks.

Reply:
Done, and squashed into a single commit.

Reply:
Would love to see it merged for easy cocoapods management!